pkg/agent/controller/networkpolicy/cache.go

```diff
@@ -113,7 +115,7 @@ func (r *CompletedRule) String() string {
 
 // isAntreaNetworkPolicyRule returns true if the rule is part of a ClusterNetworkPolicy.
 func (r *CompletedRule) isAntreaNetworkPolicyRule() bool {
-	return r.PolicyPriority != nil
+	return r.PolicyPriority != nil && r.TierPriority != nil
```

Comment:
i think it is okay to just have PolicyPriority check here..

Reply:
Changed back